pipelines/hello_world/README.md

```diff
@@ -1,16 +1,16 @@
-## Getting started with notebook pipelines
+## Getting started with AI workflow pipelines
```

Comment:
I believe `workflow` and `pipeline` are sorts of synonymous here, and I would just call it 'AI pipelines'

Reply:
Changed it  in both tutorials as requested in the title and started each tutorial with a qualification to highlight that the loose definition used in the tutorials is specific to Elyra. (Pipelines in general are not restricted to those types.)

`In Elyra, an AI pipeline, also referred to as workflow pipeline, comprises of notebook nodes or Python script nodes that are connected with each other.`